src/main/java/org/wildfly/security/ssl/CipherSuiteSelector.java

```diff
@@ -540,9 +540,14 @@ private static CipherSuitePredicate getSimplePredicateByName(final String word)
             case "AES256":      return CipherSuitePredicate.matchEncryption(Encryption.AES256, Encryption.AES256GCM);
             case "AES":         return CipherSuitePredicate.matchEncryption(Encryption.AES128, Encryption.AES128GCM, Encryption.AES256, Encryption.AES256GCM);
             case "AESGCM":      return CipherSuitePredicate.matchEncryption(Encryption.AES128GCM, Encryption.AES256GCM);
+            case "ARIA":        return CipherSuitePredicate.matchEncryption(Encryption.ARIA128, Encryption.ARIA128GCM, Encryption.ARIA256, Encryption.ARIA256GCM);
```

Comment:
Shouldn't these be in a separate commit/JIRA?  Or are they somehow related to chacha20-poly1305?

Reply:
I added them as requested by @mchoma comment in jira conversation https://issues.jboss.org/browse/ELY-1535. But yes, I should've made a separate jira for it, so I am going to correct that.